src/Microsoft.Data.SqlClient/tests/ManualTests/config.default.json

```diff
@@ -14,5 +14,10 @@
     "SupportsIntegratedSecurity": true,
     "SupportsLocalDb": false,
     "SupportsFileStream": false,
-    "UseManagedSNIOnWindows": false
+    "UseManagedSNIOnWindows": false,
+    "DNSCachingConnString": "",
+    "DNSCachingServerCR": "",
+    "DNSCachingServerTR": "",
+    "IsDNSCachingSupportedCR": false,
+    "IsDNSCachingSupportedTR": false
```

Comment:
Update BUILDGUIDE

Reply:
They will be removed as per offline discussion.